cobbler/items/item.py

```diff
@@ -721,20 +661,20 @@ def logical_parent(self) -> Any:
         return parent
 
     @property
-    def children(self) -> List["ITEM_UNION"]:
+    def children(self) -> List["Item"]:
         """
         The list of logical children of any depth.
         :getter: An empty list in case of items which don't have logical children.
         :setter: Replace the list of children completely with the new provided one.
         """
-        results: List[Any] = []
+        results: List["Item"] = []
         list_items = self.api.get_items(self.COLLECTION_TYPE)
         for obj in list_items:
-            if obj.get_parent == self._name:
-                results.append(obj)
+            if obj.get_parent == self._name:  # type: ignore
+                results.append(obj)  # type: ignore
```

Comment:
I will not do this since this will be refactored for `InheritableItem` in the follow-up PR.

Reply:
I'm OK with that approach, my only worry is that we'll forget about these since we have type: ignore comments that will either mask future errors, or be obsolete. 

I don't think we have to create an issue for two `# type: ignore` comments, maybe a `TODO: ` comment above this would suffice so that next time we touch the code, we can remove this?

Reply:
TODO added. Will be pushed.